source/_static/enmap.org/landingpage.html

```diff
@@ -119,20 +119,16 @@ <h2>Links</h2>
                               <li>Documentation <a
                                   href="https://enmap-box.readthedocs.io">https://enmap-box.readthedocs.io</a></li>
                               <li>Source Code <a
-                                  href="https://github.com/hu-geomatics/enmap-box">https://github.com/hu-geomatics/enmap-box</a>
+                                  href="https://github.com/Florian-Katerndahl/enmap-box">https://github.com/Florian-Katerndahl/enmap-box</a>
                               </li>
                               <li>QGIS Plugin<a
                                   href="http://plugins.qgis.org/plugins/enmapboxplugin/">http://plugins.qgis.org/plugins/enmapboxplugin/</a>
                               </li>
                               <li>Issue Tracker <a
                                   href="https://bitbucket.org/hu-geomatics/enmap-box/issues">https://bitbucket.org/hu-geomatics/enmap-box/issues</a>
                               </li>
-                              <li>Changelog <a
```

Comment:
Instead removing the entire `<li>` tag, can you replace
https://github.com/hu-geomatics/enmap-box/src/develop/CHANGELOG by
https://github.com/EnMAP-Box/enmap-box/blob/main/CHANGELOG.rst instead?